src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
+import React, { FormEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo.type';
+import { TodoList } from './components/TodoList';
+
+const todosWithUser: Todo[] = todosFromServer.map(todo => {
+  const curUser = usersFromServer.find(user => todo.userId === user.id);
+
+  return {
+    ...todo,
+    user: curUser as Todo['user'],
+  };
+});
+
+export const App: React.FC = () => {
+  const [input, setInput] = useState('');
+  const [select, setSelect] = useState('0');
+  const [errors, setErrors] = useState<{ [key: string]: string }>({});
```

Comment:
```suggestion
  type Field = 'title' | 'userId'

  const [errors, setErrors] = useState<Record<Field, string>>({});
```
Define what fields error state has for example with union or enum type